tsconfig.json

```diff
@@ -20,7 +20,8 @@
     "types": ["node", "jest-extended", "expect-more-jest"],
     "allowJs": true,
     "checkJs": true,
-    "importsNotUsedAsValues": "error"
+    "importsNotUsedAsValues": "error",
+    "skipLibCheck": true
```

Comment:
no

Reply:
This has been reverted as I no longer am using the atlaskit packages